app/config/index.js

```diff
@@ -0,0 +1,3 @@
+"use strict";
+
+module.exports.Config = require('./config');
```

Comment:
Easier to just consume the file with one more path instead of having to re-export every new file to remove some of the path when requiring the module don't you think? Harder to maintain